Jenkinsfile

```diff
@@ -12,9 +12,10 @@ ansiColor('xterm') {
             sh "ansible-playbook jenkins.yml --syntax-check"
         }
         stage('Build') {
+            // add aws_access_keys on your aws jenkins instance
             withCredentials([usernamePassword(credentialsId: 'aws_access_keys', usernameVariable: 'AWS_ACCESS_KEY', passwordVariable: 'AWS_SECRET_KEY')]) {
             // Run the packer build
-                sh "packer build -var 'aws_region=eu-west-1' jenkins.json"
```

Comment:
Ideally this should be pulled from an environment variable set on your Jenkins instance, just like the AWS creds